src/lib.rs

```diff
@@ -148,7 +148,7 @@
 //!     }
 //!
 //!     // Define additional `ErrorKind` variants. The syntax here is
-//!     // the same as `quick_error!`, but the `from()` and `cause()`
+//!     // the same as `impl_error_chain_kind!`, but the `from()` and `cause()`
```

Comment:
So it is well documented in src/error_chain.rs#57 but that doc comment doesn't seem to appear on any of the generated docs.  I'm not sure how to link to this documentation.